src/server/req_manager.c

```diff
@@ -1677,6 +1682,68 @@ mgr_server_unset(struct batch_request *preq)
 		if (server.sv_attr[SVR_ATR_DefaultChunk].at_flags & ATR_VFLAG_MODIFY) {
 			(void)deflt_chunk_action(&server.sv_attr[SVR_ATR_DefaultChunk], (void *)&server, ATR_ACTION_ALTER);
 		}
+		/* Now set the default values on some of the unset attributes */
+		plist = (svrattrl *)GET_NEXT(preq->rq_ind.rq_manager.rq_attr);
+		for (plist = (svrattrl *)GET_NEXT(preq->rq_ind.rq_manager.rq_attr);
+			plist != NULL; plist = (struct svrattrl *)GET_NEXT(plist->al_link)) {
+			if (strcasecmp(plist->al_name, ATTR_logevents) == 0) {
+				char dflt_log_event[64];
+				sprintf(dflt_log_event, "%d", SVR_LOG_DFLT);
+				set_attr_svr(&(server.sv_attr[(int)SRV_ATR_log_events]), &svr_attr_def[(int) SRV_ATR_log_events],
+					     dflt_log_event);
+			}
+			else if (strcasecmp(plist->al_name, ATTR_mailfrom) == 0)
+				set_attr_svr(&(server.sv_attr[(int)SRV_ATR_mailfrom]),
+					    &svr_attr_def[(int)SRV_ATR_mailfrom], "adm");
+			else if (strcasecmp(plist->al_name, ATTR_queryother) == 0)
+				set_attr_svr(&(server.sv_attr[(int)SRV_ATR_query_others]),
+					    &svr_attr_def[(int)SRV_ATR_query_others], "TRUE");
+			else if (strcasecmp(plist->al_name, ATTR_schediteration) == 0)
```

Comment:
didn't we already handle sched iteration in the while block before in this function?  Also, should these 2 loops be combined?